test.sh

```diff
@@ -6,5 +6,5 @@ set -e
 if [[ ${CI} != "true" || (${NODE_ENV} = "test" && ${TRAVIS_PULL_REQUEST} != "false") ]]; then
 
   bundler --version
```

Comment:
I think this is now redundant with the UI tests being 100% JS

Reply:
As far as I understand this is the logic linked with the `travis.yml` file where we specify envs `NODE_ENV=test` so we need to keep it. We use similar in `setup_test_environment.sh`.

Reply:
I might be wrong, but Bundler is a Ruby thing so doesn't seem like it will still be needed since we are removing the UI tests' dependency on Ruby. 
Though TBH all this line really does is print out what version of Bundler is installed on an environment, if any.

Reply:
I understood that the comment was about line number 6 instead of 8. Removed - a42d0a6